cmake/configure/configure_2_trilinos.cmake

```diff
@@ -192,13 +192,14 @@ MACRO(FEATURE_TRILINOS_CONFIGURE_EXTERNAL)
   SET(DEAL_II_EXPAND_TRILINOS_MPI_BLOCKVECTOR "TrilinosWrappers::MPI::BlockVector")
   SET(DEAL_II_EXPAND_TRILINOS_MPI_VECTOR "TrilinosWrappers::MPI::Vector")
 
+  ENABLE_IF_SUPPORTED(TRILINOS_CXX_FLAGS "-Wno-extra")
```

Comment:
On 04/15/2015 11:33 AM, Matthias Maier wrote:
> So you want this flag to be set regardless of the Trilinos version?
> Yes. For the time being.

No, I don't care. I was just wondering whether that's what you intended.